Description: Remove leading whitespace from texinfo directory entry
 The leading whitespace makes Emacs' info menu not work as expected,
 at least With newer version or combinations of texinfo and/or yodl.
Author: Bruce Stephens <bruce.stephens@isode.com>
Bug-Debian: http://bugs.debian.org/711516
Forwarded: http://www.zsh.org/mla/workers/2013/msg00520.html

Index: zsh/Doc/ztexi.yo
===================================================================
--- zsh.orig/Doc/ztexi.yo	2013-06-13 02:31:40.743480301 +0200
+++ zsh/Doc/ztexi.yo	2013-06-13 02:32:50.236628923 +0200
@@ -71,7 +71,7 @@
   NOTRANS(@ifinfo)+NL()\
   NOTRANS(@dircategory Utilities)+NL()\
   NOTRANS(@direntry)+NL()\
-  NOTRANS(     * ZSH: (zsh).                     The Z Shell Manual.)+NL()\
+  NOTRANS(* ZSH: (zsh).                     The Z Shell Manual.)+NL()\
   NOTRANS(@end direntry)+NL()\
   NOTRANS(@end ifinfo)+NL()\
 )
